### ISSUE TITLE
Unable to install any packages. 

### ISSUE THREAD

**Issue by murrelljm99** [fireeye/commando-vm#197]
I am attempting to install Commando VM inside a fresh installation of Windows 10 Home 2004. I am receiving errors related to #123. The install script will fail to install all of the packages outside of the the initial configuration. However, if I install any of the packages via cinst -y it is successful. Is there a way to resolve this issue with the script?

I tried to install via: 

.\install.ps1 -profile_file .\profiles\full.json -password password -nochecks 1

**Comment by day1player** [comment]
@murrelljm99 if I am understanding your issue that is something I have not been able to figure out. That is why that issue will probably remain up for a while. I also put a notice in the initial install about reading up on that issue. Unfortunately when that happens the only way I know how to reliably fix it is to take care of those packages manually.

**Comment by murrelljm99** [comment]
Well the issue occurs during the entirety of the install, no packages will install outside of the first few. The immediate answer is to install most of the packages manually, but that somewhat defeats the purpose of the script. I'm just curious as to why it appears to work for some people and not for others. I've gone ahead and installed some manually, but it's an interesting issue!

**Comment by day1player** [comment]
> that somewhat defeats the purpose of the script

I completely agree, its unfortunate. I have worked with myget to see if they were throttling downloads and it doesnt appear that they are. Boxstarter/chocolatey dont appear to be interfering either. Its a very puzzling mystery to me as to why packages do that but usually its no more than 3-4 and usually only happens at the end of an install..

What I like to do when this happens is keep the install up and running and when I see one fail, I open a prompt and install manually then and there. This will help decrease total install time and then you wont have to install each package manually, only the ones that fail.

**Comment by murrelljm99** [comment]
I just realized I referenced the wrong issue in my initial response, I meant error codes relating to #122, not #123. My apologies. I saw the manual install as a temporary workaround. I didn't bother with it though since it was almost every package in the json that failed (unable to find package in the sources provided). The only package that was found with the sources was with the commando vm config, but then that errored out in relation to #123 where it took forever to download and then fails. Sorry for the mix up, hope this better clarifies the issue.
